pkg/printers/internalversion/describe.go

```diff
@@ -2826,6 +2827,124 @@ func (p *PodDisruptionBudgetDescriber) Describe(namespace, name string, describe
 	})
 }
 
+type PodSecurityPolicyDescriber struct {
+	clientset.Interface
+}
+
+func (d *PodSecurityPolicyDescriber) Describe(namespace, name string, describerSettings printers.DescriberSettings) (string, error) {
+	psp, err := d.Extensions().PodSecurityPolicies().Get(name, metav1.GetOptions{})
+	if err != nil {
+		return "", err
+	}
+	return tabbedString(func(out io.Writer) error {
+		w := NewPrefixWriter(out)
+		w.Write(LEVEL_0, "Name:\t%s\n", psp.Name)
+		printLabelsMultiline(w, "Labels", psp.Labels)
+		printAnnotationsMultiline(w, "Annotations", psp.Annotations)
+
+		w.Write(LEVEL_0, "\nSettings:\n")
+
+		w.Write(LEVEL_0, "  Allow Privileged:\t%t\n", psp.Spec.Privileged)
```

Comment:
As far I understand you shouldn't do alignment manually, but use `LEVEL_0`, `LEVEL_1`, etc instead.